.kokoro/generate-docs.sh

```diff
@@ -55,6 +57,14 @@ for package in $(echo "${python_bucket_items}" | cut -d "-" -f 5- | rev | cut -d
   tar -zxvf ${tarball}
   repo=$(cat docs.metadata | grep "github_repository:" | cut -d "\"" -f 2 | cut -d "/" -f 2)
 
+  # If the pacakage is part of the monorepo, we'll process this later.
+  if [[ "${repo}" == "google-cloud-python" ]]; then
+    monorepo_packages="$(cat docs.metadata | grep "distribution_name" | cut -d "\"" -f 2) ${monorepo_packages}"
```

Comment:
Why the `${monorepo_packages}` at the end?

Reply:
To continue adding to the string to be processed later as I'm keeping it in a string format, which then gets used at line 172+ to process monorepo-specific packages.

Reply:
Didn't know you could also use `+=` in bash as well. Updated to append with a whitespace at the end.